### PR TITLE
DOCSUP-6295: Updated isIPv4String and isIPv6String functions documentation

### PR DESCRIPTION
**Pull request by gyuton**
I hereby agree to the terms of the CLA available at: https://yandex.ru/legal/cla/?lang=en

Changelog category:

- Documentation for #19489

### CODE REVIEW

File: docs/en/sql-reference/functions/ip-address-functions.md [hunk 1]
```diff
@@ -265,32 +265,86 @@ SELECT toIPv6('127.0.0.1')
 └─────────────────────┘
 ```
 
-## isIPv4String
+## isIPv4String {#isIPv4String}
```
Comment (olgarev) [inline docs/en/sql-reference/functions/ip-address-functions.md:268]:
```suggestion
## isIPv4String {#isipv4string}
```

File: docs/en/sql-reference/functions/ip-address-functions.md [hunk 2]
```diff
@@ -265,32 +265,86 @@ SELECT toIPv6('127.0.0.1')
...
-SELECT isIPv4String('127.0.0.1')
+**Syntax**
+
+```sql
+isIPv4String(string)
 ```
 
+**Parameters**
+
+-   `string` — String. [String](../../sql-reference/data-types/string.md).
```
Comment (olgarev) [inline docs/en/sql-reference/functions/ip-address-functions.md:280]:
```suggestion
-   `string` — IP address. [String](../../sql-reference/data-types/string.md).
```

File: docs/en/sql-reference/functions/ip-address-functions.md [hunk 3]
```diff
@@ -265,32 +265,86 @@ SELECT toIPv6('127.0.0.1')
...
+┌─isIPv4String('0.0.0.0')─┐
+│                       1 │
+└─────────────────────────┘
+┌─isIPv4String('Hello')─┐
+│                     0 │
+└───────────────────────┘
 ```
 
-## isIPv6String
+## isIPv6String {#isIPv4String}
```
Comment (olgarev) [inline docs/en/sql-reference/functions/ip-address-functions.md:309]:
```suggestion
## isIPv6String {#isipv6string}
```

File: docs/en/sql-reference/functions/ip-address-functions.md [hunk 4]
```diff
@@ -265,32 +265,86 @@ SELECT toIPv6('127.0.0.1')
...
+
+**Syntax**
+
+```sql
+isIPv6String(string)
+```
+
+**Parameters**
+
+-   `string` — String. [String](../../sql-reference/data-types/string.md).
```
Comment (olgarev) [inline docs/en/sql-reference/functions/ip-address-functions.md:321]:
```suggestion
-   `string` — IP address. [String](../../sql-reference/data-types/string.md).
```

File: docs/en/sql-reference/functions/ip-address-functions.md [hunk 5]
```diff
@@ -265,32 +265,76 @@ SELECT toIPv6('127.0.0.1')
...
-┌─isIPv4String('127.0.0.1')─┐
-│                         1 │
-└───────────────────────────┘
+┌─isIPv4String('0.0.0.0')─┐
+│                       1 │
+└─────────────────────────┘
 ```
 
-## isIPv6String
+## isIPv6String {#isipv4string}
```
Comment (olgarev) [inline docs/en/sql-reference/functions/ip-address-functions.md:304]:
```suggestion
## isIPv6String {#isipv6string}
```

File: docs/ru/sql-reference/functions/ip-address-functions.md [hunk 6]
```diff
@@ -243,4 +243,76 @@ SELECT
...
+
+Результат:
+
+``` text
+┌─isIPv4String('0.0.0.0')─┐
+│                       1 │
+└─────────────────────────┘
+```
+
+## isIPv6String {#isipv4string}
```
Comment (olgarev) [inline docs/ru/sql-reference/functions/ip-address-functions.md:282]:
```suggestion
## isIPv6String {#isipv6string}
```

**Comment by gyuton** [comment]
@vdimir , Hi!
Can you review this, please?

**Review by vdimir — APPROVED** [review]
I've only added more examples with different strings.

Thank you for you PR.

**Comment by gyuton** [comment]
> 
> 
> I've only added more examples with different strings.
> 
> Thank you for you PR.

@vdimir, you're welcome!

**Pull request merged**